<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite start="1638725286162" stop="1638725308016" version="1.5.4" xmlns:ns2="urn:model.allure.qatools.yandex.ru">
    <name>Default suite : Default test</name>
    <title>Default suite : Default test</title>
    <test-cases>
        <test-case start="1638725286225" stop="1638725307901" status="failed">
            <name>verifyConfirmationId</name>
            <failure>
                <message>ComparisonFailure: null expected:&lt;1638725307679&gt; but was:&lt;null&gt;</message>
                <stack-trace>junit.framework.ComparisonFailure: null expected:&lt;1638725307679&gt; but was:&lt;null&gt;&#13;
	at junit.framework.Assert.assertEquals(Assert.java:81)&#13;
	at junit.framework.Assert.assertEquals(Assert.java:87)&#13;
	at com.maersk.com.maerskint.ExcriceMaersk.verifyConfirmationId(ExcriceMaersk.java:65)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:90)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:55)&#13;
	at java.lang.reflect.Method.invoke(Method.java:508)&#13;
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)&#13;
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)&#13;
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)&#13;
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)&#13;
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)&#13;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)&#13;
	at org.testng.TestRunner.privateRun(TestRunner.java:648)&#13;
	at org.testng.TestRunner.run(TestRunner.java:505)&#13;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)&#13;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)&#13;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)&#13;
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)&#13;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)&#13;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)&#13;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)&#13;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)&#13;
	at org.testng.TestNG.runSuites(TestNG.java:1049)&#13;
	at org.testng.TestNG.run(TestNG.java:1017)&#13;
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)&#13;
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)&#13;
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)&#13;
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default suite"/>
                <label name="testGroup" value="Default test"/>
                <label name="testClass" value="com.maersk.com.maerskint.ExcriceMaersk"/>
                <label name="testMethod" value="verifyConfirmationId"/>
                <label name="host" value="LAPTOP-DQDU3SMA"/>
                <label name="thread" value="64112@LAPTOP-DQDU3SMA.main(1)"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
